toolkit/featureforms/build.gradle.kts

```
@@ -50,6 +51,27 @@ android {
     }
 }
 
+apiValidation {
+    ignoredClasses.add("com.arcgismaps.toolkit.featureforms.BuildConfig")
+    // todo: remove when this is resolved https://github.com/Kotlin/binary-compatibility-validator/issues/74
+    // compose compiler generates public singletons for internal compose functions. this may be resolved in the compose
+    // compiler.
+    val composableSingletons = listOf(
+        "com.arcgismaps.toolkit.featureforms.components.base.ComposableSingletons\$BaseTextFieldKt",
+        "com.arcgismaps.toolkit.featureforms.components.codedvalue.ComposableSingletons\$ComboBoxFieldKt",
+        "com.arcgismaps.toolkit.featureforms.components.codedvalue.ComposableSingletons\$RadioButtonFieldKt",
+        "com.arcgismaps.toolkit.featureforms.components.datetime.ComposableSingletons\$DateTimeFieldKt",
+        "com.arcgismaps.toolkit.featureforms.components.datetime.picker.ComposableSingletons\$DateTimePickerKt",
+        "com.arcgismaps.toolkit.featureforms.components.datetime.picker.date.ComposableSingletons\$DatePickerKt",
+        "com.arcgismaps.toolkit.featureforms.components.datetime.picker.time.ComposableSingletons\$TimePickerKt",
+        "com.arcgismaps.toolkit.featureforms.components.formelement.ComposableSingletons\$AttachmentFormElementKt",
+        "com.arcgismaps.toolkit.featureforms.components.formelement.ComposableSingletons\$GroupElementKt"
+    )
+    
+    ignoredClasses.addAll(composableSingletons)
```

```suggestion
    ignoredPackages.add("com.arcgismaps.toolkit.featureforms.components")
```
I think the entire package can be ignored.

---

> I think the entire package can be ignored. It might be a little tedious to keep track of all the internal compose functions.

@kaushikrw I'd like to discuss this point a bit offline.

---

As discussed, this can go in now and we will later move all the internal components into an `internal` package for feature forms and ignore that package.